politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -148,12 +151,15 @@ const usageMsg = `politeiawww_dbutil usage:
           Migrate a LevelDB user database to CockroachDB
           Required DB flag : None
           Args             : None
-
-     -verifyidentities
+    -verifyidentities
           Verify a user's identities do not violate any politeia rules. Invalid
           identities are fixed.
           Required DB flag : -cockroachdb
-          Args             : <username>
+		  Args             : <username>
```

Comment:
Looks like the spaces were replaces with tabs here and is throwing off the alignment.